### PR TITLE
Members only feature

### PR DESCRIPTION
**Pull request by GabrielJSuarez**
Create members Only project using

- DEVISE
- GUARD
- BOOTSTRAP 4
- BETTER ERRORS
- SIMPLE FORMS

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
-Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  resources :posts
+  devise_for :users
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'posts#index'
```
Comment (acushlakoncept) [inline config/routes.rb:7]: Making your application routes to use `resources :posts, only: [:new, :create, :index]` is part of the project requirements ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)) Kindly update accordingly.

File: app/helpers/posts_helper.rb [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# rubocop:disable all
+
```
Comment (acushlakoncept) [inline app/helpers/posts_helper.rb:2]: RuboCop cops are meant to catch bad code and enforce best practices and the rubocop.yml file provided by [Microverse linters](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) repo doesn't enforce strict rules.

By disabling RuboCop rules, you are suppressing the potential warnings you would be getting, missing out on making your Ruby code more idiomatic. (See [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

Kindly remove line 1 and try to fix the rubocop linter error.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,67 @@
...
+$ git clone https://github.com/GabrielJSuarez/members_only_the_odin_project.git
+
+Go to your directory
+
+$ `cd members_only_the_odin_project
+
+Run in terminal [You must have Ruby installed on your system.]
+
+$ `ruby bundle install`
+
```
Comment (acushlakoncept) [inline README.md]: add in `ruby` will throw an error kindly update to just `bundle install`

File: README.md [hunk 4]
```diff
@@ -1,24 +1,67 @@
...
-* ...
+# README
+# Project: Members only!
+
+# Build with: 
+
+- Bootstrap 4 css implementation for Rails
+- Simple Forms Gem
+- Better Erros Gem
+- Guard Gem
```
Comment (acushlakoncept) [inline README.md]: You have a little typo here...`Better **Errors** ...` missing `r`

File: README.md [hunk 5]
```diff
@@ -1,24 +1,67 @@
...
+$ `cd members_only_the_odin_project
+
+Run in terminal [You must have Ruby installed on your system.]
+
+$ `ruby bundle install`
+
+Install the Bootstrap Dependencies Locally to show the bootstrap styling
+
+$ `yarn add bootstrap@4.3.1 jquery popper.js`
+
```
Comment (acushlakoncept) [inline README.md:28]: [OPTIONAL], running `yarn add` alone does the job

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Hello @GabrielJSuarez  sorry about the last review, Your project looks really amazing. The UI is excellent and you have even gone ahead to implement extra features beyond the project scope, that is applaudable, I love it. You didn't need to open another PR though.

However, there are still a few changes that you need to update for this project to be approved.

Kindly refer to the comments on your codebase below.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by trekab — APPROVED** [review]
### Approved! :tada: 
Well done implementing the functional requirements for this project, @GabrielJSuarez! There are just a few optional issues you could consider though:
- [ ] __[Optional]__ Clean up your repo by adding the `.idea` folder to your `.gitignore` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] __[Optional]__ Your use of the `rubocop:disable all` in the `posts_helper.rb` file is rather extreme. Like the last reviewer advised, disabling linting this way will just permit committing bad code to your repo.

_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**